src/components/ImportExport.component.js

```diff
@@ -75,6 +75,23 @@ class ImportExport extends React.Component {
         }
     };
 
+    exportEmptyTemplate = () => {
+        const { allColumns } = this.props;
+        this.setState({ isProcessing: true });
+
+        try {
+            const labeledColumns = allColumns.map(column => column.text);
+            const blob = new Blob([labeledColumns], { type: "text/plain;charset=utf-8" });
+            const datetime = moment().format("YYYY-MM-DD_HH-mm-ss");
+            const filename = `empty-user-template-${datetime}.csv`;
+            FileSaver.saveAs(blob, filename);
+            snackActions.show({ message: `${this.t("table_exported")}: ${filename}` });
+        } finally {
+            this.closeMenu();
+            this.setState({ isProcessing: false });
+        }
+    };
+
```

Comment:
The common code with `exportToCsvAndSave` should be abstracted somehow. There are two typical approaches: 1) abstract the full method passing the particularities to the function (even on the form of functions, it behavious must be abstracted), or 2) leave them more independent and abstract the pieces (more repetition but more versatile and usually more clear). There are pros and cons for all approaches. Check bfbc7bf for my (2) approach, I've abstracted only the specific `saveCsv` code.